### PR TITLE
Fixup Emulator::GetBackgroundPicturePath()

### PR DESCRIPTION
**Pull request by elad335**
Was not detecting some images for disc-based games with updates.

### CODE REVIEW

File: rpcs3/Emu/System.cpp [hunk 1]
```diff
@@ -739,18 +739,23 @@ std::string Emulator::GetBackgroundPicturePath() const
...
-		disc_dir.empty() ? (disc_dir + "/PIC2.PNG") : disc_dir,
-		disc_dir.empty() ? (disc_dir + "/PIC3.PNG") : disc_dir,
+
+		!disc_dir.empty() ? (disc_dir + "/PIC0.PNG") : disc_dir,
+		!disc_dir.empty() ? (disc_dir + "/PIC1.PNG") : disc_dir,
+		!disc_dir.empty() ? (disc_dir + "/PIC2.PNG") : disc_dir,
+		!disc_dir.empty() ? (disc_dir + "/PIC3.PNG") : disc_dir,
+
+		!disc_dir.empty() ? (disc_dir + "/ICON0.PNG") : disc_dir,
+		m_sfo_dir + "/ICON0.PNG",
```
[the remark this answers is not in the document]
Comment (elad335) [reply rpcs3/Emu/System.cpp:758]: It simplifies the logic for found_pic_in_sfo_dir

[the remark this answers is not in the document]
Comment (Megamouse) [reply rpcs3/Emu/System.cpp:756]: How

[the remark this answers is not in the document]
Comment (Megamouse) [reply rpcs3/Emu/System.cpp:756]: But why is the icon0 disc dir in front of the SFO dir?

[the remark this answers is not in the document]
Comment (elad335) [reply rpcs3/Emu/System.cpp:756]: Its outdated code, it doesn't show this diff window.

**Pull request merged**